berkeley-function-call-leaderboard/model_handler/yi_handler.py

```diff
@@ -0,0 +1,81 @@
+from model_handler.handler import BaseHandler
+from model_handler.model_style import ModelStyle
+from model_handler.utils import (
+    convert_to_tool,
+    convert_to_function_call,
+    augment_prompt_by_languge,
+    language_specific_pre_processing,
+)
+from model_handler.constant import GORILLA_TO_OPENAPI
+from openai import OpenAI
+import os, time, json
+
+
+class YiHandler(BaseHandler):
+    def __init__(self, model_name, temperature=0.0, top_p=1, max_tokens=1000) -> None:
+        super().__init__(model_name, temperature, top_p, max_tokens)
+        self.model_style = ModelStyle.OpenAI
+        self.base_url = "https://api.lingyiwanwu.com/v1"
```

Comment:
Hey @fantasist and @luojy95 thanks for your PR. I was wondering if this is a good idea to have your private endpoint since others wont be able to access this? Can you either provide a hosted end point that is GA or a local inference script?

Reply:
This is the public API endpoint for Chinese users. I sent an API key to @HuanzhiMao over Discord. Wish you guys a smooth testing experience.

Reply:
I confirmed with @fantasist that this endpoint is not a personal/private endpoint and it will be permanent. Their company website is https://www.lingyiwanwu.com/en